doc/source/timedeltas.rst

```diff
@@ -363,13 +363,44 @@ or ``np.timedelta64`` objects. Passing ``np.nan/pd.NaT/nat`` will represent miss
    pd.TimedeltaIndex(['1 days', '1 days, 00:00:05',
                      np.timedelta64(2,'D'), datetime.timedelta(days=2,seconds=2)])
 
-Similarly to ``date_range``, you can construct regular ranges of a ``TimedeltaIndex``:
+Similar to ``date_range``, you can construct regular ranges of a ``TimedeltaIndex``
+using ``timedelta_range``.  The default frequency for ``timedelta_range`` is 
+calendar day:
+
+.. ipython:: python
+
+   pd.timedelta_range(start='1 days', periods=5)
+
+Various combinations of ``start``, ``end``, and ``periods`` can be used with
+``timedelta_range``:
+
+.. ipython:: python
+
+   pd.timedelta_range(start='1 days', end='5 days')
+
+   pd.timedelta_range( end='10 days', periods=4)
```

Comment:
Small nit: There's an extra space between the left parenthesis and end `pd.timedelta_range( end=...)`

Reply:
thanks, done